README.md

```diff
@@ -1,116 +1,267 @@
+
+You can find more information in the Results section of our paper [Are You Using Retentive Networks?](./301R_Retnet_Paper.pdf).
+
+## Acknowledgments
+
+We extend our heartfelt gratitude to the following individuals and institutions for their invaluable contributions to this project:
+
+**Nancy Fulda**: Our esteemed instructor, whose guidance and insights have significantly shaped the direction and execution of this research.
+
+**BYU Office of Research Computing**: For providing the computational resources and support that were instrumental in conducting our experiments and analyses.
+
+**Authors of the Original RetNet Paper**: For their pioneering work in the field of deep learning, which inspired our research and provided a solid base for our explorations into RetNet and Transformers.
+
+**Microsoft TorchScale Team**: For developing and maintaining the TorchScale framework, which served as the foundational architecture for our project, enabling us to push the boundaries of what's possible in deep learning research.
+
+## Citations
+
+This can also be found on the citation section or our paper [Are You Using Retentive Networks?](./301R_Retnet_Paper.pdf).
+
+```
+@misc{eval-harness,
+  author       = {Gao, Leo and Tow, Jonathan and Abbasi, Baber and Biderman, Stella and Black, Sid and DiPofi, Anthony and Foster, Charles and Golding, Laurence and Hsu, Jeffrey and Le Noac'h, Alain and Li, Haonan and McDonell, Kyle and Muennighoff, Niklas and Ociepa, Chris and Phang, Jason and Reynolds, Laria and Schoelkopf, Hailey and Skowron, Aviya and Sutawika, Lintang and Tang, Eric and Thite, Anish and Wang, Ben and Wang, Kevin and Zou, Andy},
+  title        = {A framework for few-shot language model evaluation},
+  month        = 12,
+  year         = 2023,
+  publisher    = {Zenodo},
+  version      = {v0.4.0},
+  doi          = {10.5281/zenodo.10256836},
+  url          = {https://zenodo.org/records/10256836}
+}
+
+
 ```
+```
+@article{retnet,
+  author={Yutao Sun and Li Dong and Shaohan Huang and Shuming Ma and Yuqing Xia and Jilong Xue and Jianyong Wang and Furu Wei},
+  title     = {Retentive Network: A Successor to {Transformer} for Large Language Models},
+  journal   = {ArXiv},
+  volume    = {abs/2307.08621},
+  year      = {2023}
+}
 
-In `config.yaml`, you can have:
 
-```yaml
-learning_rate: 0.001
-batch_size: 64
-num_layers: 4
-embed_dim: 256
+```
+```
+@article{torchscale,
+  author    = {Shuming Ma and Hongyu Wang and Shaohan Huang and Wenhui Wang and Zewen Chi and Li Dong and Alon Benhaim and Barun Patra and Vishrav Chaudhary and Xia Song and Furu Wei},
+  title     = {{TorchScale}: {Transformers} at Scale},
+  journal   = {CoRR},
+  volume    = {abs/2211.13184},
+  year      = {2022}
+}
```

Comment:
This whole section should be reduced and simplified. These citations should be included in the paper and deleted from the README. The purpose of this section is to give our citation once published for others to cite **us**. For now, this should be something simple stating that we're still waiting to be published and our citation will be given soon.

Reply:
I agree, most of this should be cut

Reply:
I fixed this.